vocode/telephony.py

```diff
@@ -0,0 +1,43 @@
+import logging
+import requests
+from vocode.models.agent import InformationRetrievalAgentConfig, LLMAgentConfig
+from vocode.models.telephony import CallEntity, CreateCallRequest
+from . import api_key
+
+BASE_URL = "https://0a7e-136-24-82-111.ngrok.io"
+
+
+class Telephony:
+    def __init__(self, logger: logging.Logger = None) -> None:
+        self.logger = logger
+
+    def create_call(self, request: CreateCallRequest):
+        request_data = request.dict()
+
+        url = f"{BASE_URL}/create_outbound_call?key={api_key}"
```

Comment:
can we change the URL to `phonecall` instead of `create_outbound_call`? looks nicer